tool/simulation/change_parameters.py

```diff
@@ -0,0 +1,172 @@
+"""
+This is the small script of changing the parameters of Input__* files.
+
+How to use it:
+  1. Set up the files and the parameters you want ot change.
+    a. The file is set to be a `File()` class. The `File` takes four inputs: file name, constant
+       parameters, changing parameters, and flag file or not. Please check out the `Classes` section
+       for the detail of inputs.
+    b. Wrap all the `File` classes to a list called `files`.
+
```

Comment:
Maybe explain here that there are two different file formats:
Files with two columns without header where the first column specifies the option's name and files with N columns where the header starts with a "#" and the following column names in the header specify the options' names.

Reply:
Updated!